spec/controllers/reports_controller_spec.rb

```diff
@@ -33,19 +33,46 @@
     end
   end
 
+  context 'GET /reports/near/:lat/:long.json' do
+    before do
+      Report.destroy_all
+      3.times do
+        Report.create! valid_create_params
+        Report.create! far_away_create_params
+      end
+    end
+
+    it 'lists only reports near location' do
+      get :index, latitude: 56.878333, longitude: 14.809167
+      expect(json_response.size).to eq(3)
```

Comment:
It really does not make sense to split it into two different expects here. We are basically checking one thing here (that it returns the three reports that are correct) - and setting up an extra block for this would make no sense, just add noise.

See example here http://betterspecs.org/#single